### ISSUE TITLE
Support for Orleans 2.0 - Released

### ISSUE THREAD

**Issue by dsarfati** [OrleansContrib/OrleansDashboard#114]
I just migrated my project from 2.0-beta3 to the recently released 2.0 👏 . I get an error when trying to start my silo using Dashboard 2.0.0-rc2a. Has anyone been able to successfully add dashboard to 2.0?

```
Error while running silos
System.TypeLoadException: Method 'Invoke' in type 'OrleansDashboard.GrainProfiler' from assembly 'OrleansDashboard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
   at Orleans.ServiceCollectionExtensions.UseDashboard(ISiloHostBuilder builder, Action`1 configurator)
   at Zapic.Service.Host.Program.<StartSilo>d__6.MoveNext() in /Users/daniel/Dev/Zapic-Platform/backend/src/Zapic.Service.Host/Program.cs:line 145
```

**Comment by richorama** [comment]
@dsarfati this should be fixed by release 2.0.0 (PR #116)

**Comment by dsarfati** [comment]
Great! Thanks!
